app/apollo/resolvers/subscription.js

```diff
@@ -328,6 +329,9 @@ const subscriptionResolvers = {
         if( clusterId ) validateString( 'clusterId', clusterId );
         tags.forEach( value => { validateString( 'tags', value ); } );
 
+        await validAuth(me, org_id, ACTIONS.CREATE, TYPES.SUBSCRIPTION, queryName, context, [name]);
+        logger.info({req_id, user, org_id, name, channel_uuid, version_uuid}, `${queryName} validating - authorized`);
+
```

Comment:
Shouldn't READ auth to the specified Groups and specified Config (channel) also be verified?

Reply:
Fixed